### DISCUSSION TITLE
extraInputAttributes on select

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#1344] [General]
Just a curio really. I noticed that the `Filament\Forms\Components\Select` component doesn't use the trait `HasExtraInputAttributes` but does use `HasExtraAttributes` which is used to add attributes to the html select element. Should this be changed for consistency? I just submitted a PR to clarify the use of `extraAttributes` and `extraInputAttributes` for the docs, stumbled across this and thought it may be confusing. (This may be the case with other components but not had chance to look into it more thoroughly)

**Comment by danharrin** [comment]
Sure, I've just been gradually adding support for that trait as people have requested it so a PR is appreciated. If you see any other components that need it, please add it.

**Reply by titantwentyone** [reply]
Thanks Dan. I'm stacked at the moment but will def have a look and fix what i can when I get some time.
